bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -33,12 +33,6 @@
 
 {% block body_class %}sky{% endblock %}
 
-{% block optimizely %}
```

Comment:
Nope, I thought we generally removed these blocks once no optimizely tests are run against the page. If we generally leave them in place, then I will do so here as well.